doc/manual/advanced-topics/distributed-builds.xml

```diff
@@ -112,4 +112,14 @@ running, they should use the same <envar>NIX_CURRENT_LOAD</envar>
 file.  Maybe in the future <filename>build-remote.pl</filename> will
 look at the actual remote load.</para>
 
+<para>Additionally, you may define the environment variable
+<envar>NIX_REMOTE_USE_SUBSTITUTES</envar>, which will instruct
```

Comment:
I expected the options in `nix.conf` to be directly related to Nix, not to the `remote-build` hook. But I agree that a `nix.conf` option would be better, if that is acceptable.